packages/next/src/server/web/spec-extension/revalidate.ts

```diff
@@ -12,20 +12,40 @@ import { workUnitAsyncStorage } from '../../app-render/work-unit-async-storage.e
 import { DynamicServerError } from '../../../client/components/hooks-server-context'
 
 /**
+ * @deprecated this function has been deprecated in favor of expireTag()
  * This function allows you to purge [cached data](https://nextjs.org/docs/app/building-your-application/caching) on-demand for a specific cache tag.
  *
  * Read more: [Next.js Docs: `revalidateTag`](https://nextjs.org/docs/app/api-reference/functions/revalidateTag)
  */
 export function revalidateTag(tag: string) {
-  return revalidate(tag, `revalidateTag ${tag}`)
+  return revalidate([tag], `revalidateTag ${tag}`)
```

Comment:
Maybe add console warning to inform this API is deprecated when users is called in runtime? Especially for non TS use case or legacy code user didn't notice

Reply:
Hopefully the deprecated type is enough, ideally we don't spam logs until the user has a chance to rename and since API surface is pretty close this one might be alright without